operators/understanding/olm-understanding-operatorhub.adoc

```diff
@@ -7,6 +7,12 @@ include::_attributes/common-attributes.adoc[]
 toc::[]
 
 include::modules/olm-operatorhub-overview.adoc[leveloffset=+1]
+
+[NOTE]
+====
+The `marketplace` Operator populates OperatorHub with a pre-defined xref:../../operators/understanding/olm-rh-catalogs.adoc#olm-rh-catalogs_olm-rh-catalogs[set of Operator catalogs]. If you disable the `marketplace` Operator during installation, OperatorHub will not be populated with these catalogs. You can xref:../../operators/admin/olm-managing-custom-catalogs.adoc#olm-creating-catalog-from-index_olm-managing-custom-catalogs[add a custom catalog source to a cluster] if you want to install Operators after disabling the `marketplace` Operator. {product-title} does not automatically update manually created catalog sources.
```

Comment:
Capabilities could also be enabled on an already running cluster. They could not be disabled once enabled. Enabling a capability is the simplest way to have OperatorHub working.

To enable a capability, cluster admin needs to edit `clusterversion` CR named `version`, and set `spec.capabilities.additionalEnabledCapabilities` list to include such capability.

Reply:
This could be done easily with following command:

```shell
$ oc patch clusterversion version --type json -p \
  '[{"op": "add", "path": "/spec/capabilities/additionalEnabledCapabilities", "value": ["marketplace"]}]'
$ oc get -o json clusterversion version | jq .status.capabilities
{
  "enabledCapabilities": [
    "marketplace"
  ],
  "knownCapabilities": [
    "baremetal",
    "marketplace",
    "openshift-samples"
  ]
}
```

Reply:
Thanks for this feedback @cardil - this information is already present in the [Cluster Capabilities](https://docs.openshift.com/container-platform/4.11/post_installation_configuration/cluster-capabilities.html#setting_additional_enabled_capabilities_cluster-capabilities) documentation. I will add a reference to it in the OperatorHub documentation.